tests/data/mock_manifests/filepath_submission_test_manifest.csv

```diff
@@ -1,3 +1,3 @@
-Filename,Sample ID,File Format,Component,Genome Build,Genome FASTA,Id,entityId
+Filename,SampleID,File Format,Component,Genome Build,Genome FASTA,Id,entityId
```

Comment:
May I ask why the change to `SampleID`?

Reply:
I updated the Sample ID column to SampleID because extra space was preventing being able to call the column by its name using the dataframe object, so it was being called with e.g. df._2. Now we can call with df.SampleID

Reply:
Yeah I'm worried this is going to break other tests...

Reply:
Hi @andrewelamb - you raise a good point, however this test is the only one that references `filepath_submission_test_manifest.csv`, which is why it's best to change it now in case future integration tests reference it
<img width="891" alt="image" src="https://github.com/user-attachments/assets/2242e96f-b2d5-44a5-88ad-1dc04493a3d5">

This is what the search for that filename came up on GitHub

Reply:
All the tests seemed to pass via CI/CD, but can we triple check "Sample ID" is not referenced in the test.

Separately, if it was being referenced as `df._2`, were you seeing that in the tests?

Reply:
Awesome thanks for checking!

Reply:
> can we triple check "Sample ID" is not referenced in the test?

I can confirm `Sample ID` is not referenced in this integration test in its current state.

>  if it was being referenced as df._2, were you seeing that in the tests?

You can view [this older commit ](https://github.com/Sage-Bionetworks/schematic/pull/1485/commits/48f10c2a69bd9a159e10ed8ad342b780262eb69d)which shows show the `Sample ID` column was referenced previously (line 60).

The `_2` is a byproduct of using `itertuples` on the dataframe. Here is a simple CSV file I made to recreate this:

<img width="532" alt="image" src="https://github.com/user-attachments/assets/8e81a404-54eb-4780-b767-cbcb340daadb">

Reply:
I want to get @linglp 's take on this before any approval. We allow users to have spaces in their manifest column names, so I want to make sure we aren't messing that up.

Reply:
Sounds good! Separately, I can add another integration test that covers for cases where the CSV files have column names with spaces.

Reply:
Apologies for the delayed response; I was out sick. we can't change the column name from Sample ID to SampleID. From what I understand, this manifest was generated using the example data model with the BulkRNA-seqAssay component data type. When I ran the manifest/generate function to create a manifest, the column headers I saw included "Sample ID" with the space. In this case, our test manifest should align with the output from manifest/generate, we need to keep the column name as is.

The issue that you refer @jaymedina , is because of this line: 
```
        manifest = helpers.get_data_frame(
            manifest_path, preserve_raw_input=False, **load_args
        )
```
If you change preserve_raw_input to True, and you also change how manifest was looped: 
```
for index, row in manifest.iterrows():
```
Does that solve the issue?

Reply:
Hi @linglp , yes switching to `iterrows()` and preserving the raw input solved this issue without needing to alter `Sample ID`, thanks!